app/src/main/java/ar/com/wolox/android/example/model/New.kt

```diff
@@ -10,5 +10,5 @@ data class New(
     @SerializedName("title") val title: String,
     @SerializedName("picture") val picture: String,
     @SerializedName("text") val text: String,
-    @SerializedName("likes") val likes: List<Int>
+    @SerializedName("likes") var likes: List<Int>
```

Comment:
Keep this as `val`, if you want to assign a different like list to a news you can use `copy(...)`